.github/workflows/deploy.yml

```
@@ -0,0 +1,46 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+      # TODO: Uses for testing this pull request only. Remove before merge!
+      - '**'
+  release:
+    types: [published]
+
+jobs:
+
+  deploy:
+    name: Deploy to WordPress.org
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          cache: 'npm'
+          node-version-file: '.nvmrc'
+
+      - name: Setup PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '5.6'
+          tools: composer:v2
+
+      - name: Install NPM dependencies
+        run: npm install
+
+      - name: Authenticate with WordPress.org SVN
+        run: svn info --non-interactive --username "${{ secrets.DEPLOY_SVN_USERNAME }}" --password "${{ secrets.DEPLOY_SVN_PASSWORD }}" https://plugins.svn.wordpress.org/two-factor/
+
+      - name: Deploy to WordPress.org SVN
+        env:
+          DEPLOY_SVN_USERNAME: ${{ secrets.DEPLOY_SVN_USERNAME }}
```

Per https://github.com/WordPress/two-factor/pull/438#discussion_r974860345 this is all I think needs changing to use the org provisioned secrets:

```suggestion
        run: svn info --non-interactive --username "${{ secrets.SVN_USERNAME }}" --password "${{ secrets.SVN_PASSWORD }}" https://plugins.svn.wordpress.org/two-factor/

      - name: Deploy to WordPress.org SVN
        env:
          DEPLOY_SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
```